MNIST/MNIST.swift

```diff
@@ -30,28 +30,46 @@ func readMNIST(imagesFile: String, labelsFile: String) -> (images: Tensor<Float>
     let images = readFile(imagesFile).dropFirst(16).map { Float($0) }
     let labels = readFile(labelsFile).dropFirst(8).map { Int32($0) }
     let rowCount = Int32(labels.count)
-    let columnCount = Int32(images.count) / rowCount
+    let imgRows : Int32 = 28, imgCols : Int32 = 28
```

Comment:
* Avoid abbreviations.
* Use a "count" suffix to denote a count.
* Remove whitespaces before colons. (See [Google Swift Style Guide - Horizontal Whitespace](https://google.github.io/swift/#horizontal-whitespace)).
```suggestion
    let rowCount: Int32 = 28, columnCount: Int32 = 28
```

Reply:
moved to imageHeight/imageWidth, removed space